File: InsuranceCompany.Client/src/app/models/PrivateClientInfo.ts
import { InsuranceRequest } from "./InsuranceRequest";

export class PrivateClientInfo {
    id : string;
    name : string;
    surname : string;
    gender : boolean;
    dateOfBirth : Date;
    personalCode: string;
    address: string;
    phoneNumber: string;
    email: string;
    insuranceRequests: InsuranceRequest[];
    // PositionClients { get; } = new List<PositionClient>();
  
    // constructor(id: string, name: string, surname: string, gender: boolean, dateOfBirth: Date, personalCode: string) {
    //     this.id = id;
    //     this.name = name;
    //     this.surname = surname;
    //     this.gender = gender;
    //     this.dateOfBirth = dateOfBirth;
    //     this.personalCode = personalCode;
    // }

    constructor(){

    }
}
  

File: InsuranceCompany.Shared/ModelDto/PrivateClientInfo.cs
using InsuranceCompany.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InsuranceCompany.Shared.ModelDto
{
    public class PrivateClientInfo
    {
        public Guid Id { get; set; }
        public string? Name { get; set; }
        public string? Surname { get; set; }
        public string PersonalCode { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? Address { get; set; }
        public bool? Gender { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public virtual ICollection<InsuranceRequest> InsuranceRequests { get; set; } = new List<InsuranceRequest>();
    }
}
